src/components/PurchaseOrder/getPOActionMenu.js

```diff
@@ -89,6 +92,17 @@ export function getPOActionMenu({
             </Icon>
           </Button>
         )}
+        <IfPermission perm="orders.item.unopen">
```

Comment:
it might be a problem if user will have this permission, but no `put` perm. Maybe it's better to check for ui permission set you've created

Reply:
It's already under `put` perm. See line 59:
`<IfPermission perm="orders.item.put">`